mpi4py.spec

```
@@ -10,13 +33,15 @@
 %endif
 
 %global with_python3 1
-%if (0%{?suse_version} >= 1500)
+%if (%{distro} == "leap15")
 %global python3_pkgversion 3
 %global _mpich_load \
- module load gnu-mpich; \
+ module_load mpich;
```

You dropped a line continuator here:
```suggestion
 module_load mpich; \
```